README.md

```diff
@@ -209,3 +209,8 @@ My name is Vanya, and I am glad to be part of the team.
 
 ![cat-forking](https://i.imgur.com/6eoCVvc.jpg)
 
+
+-----------------------------------------------------------------
+
+## Hello everybody!
+![funny cat](gif/Funny-Cat-Gifs-11.gif)
```

Comment:
It is important to have an empty line at the end of a file under CVS. Please, add one, commit, then push.